src/router/routes/job.js

```diff
@@ -247,8 +243,17 @@ router.post(
       // result is the updated job object:
       const result = await handleJob(job);
       await reportJobStatus(result);
-    } catch (err) {
-      throw err;
+    } catch (signErr) {
+      const signMessage = 'Unable to sign job';
+      logger.error(`${signMessage}, err = ${signErr.message}`);
+
+      await reportJobStatus({
```

Comment:
Do we catch error from this one?

Reply:
This reportJobStatus is probably not needed in the catch.

Reply:
Fixed.